src/mscorlib/src/System/String.cs

```diff
@@ -684,6 +685,24 @@ private unsafe string CtorReadOnlySpanOfChar(ReadOnlySpan<char> value)
             return result;
         }
 
+        public static string Create<TState>(int length, TState state, SpanAction<char, TState> action)
```

Comment:
Throw `ArgumentNullException` if `action` is null?

Reply:
Yup.